File: Problem Sets/PSet 6/Mario (less comfortable)/mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        width = height
        if height >= 1 and height <= 8:
            break
        elif height >= 10 and height <= 23:
            break

    for row in range(1, height + 1):
        # add hashes
        hashes = row
        # add spaces
        spaces = width - hashes

        # print spaces right
        print(" " * spaces, end="")

        # print hashes right
        print("#" * hashes)


if __name__ == "__main__":
    main()

File: Problem Sets/PSet 1/Mario (less comfortable)/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    for (int row = 0; row < height; row++) // Variable row is a row in the pyramid
    {
        // LEFT PYRAMID
        // Printing spaces
        for (int count = 0; count < height - row - 1; count++)
        {
            printf(" "); 
        }
        // Printing hashes
        for (int count = 0; count < row + 1; count++)
        {
            printf("#");
        }
        // New line
        printf("\n");
    }   
}
